css/liveblog.css

```diff
@@ -492,3 +492,170 @@ a.liveblog-form-entry::-webkit-input-placeholder {
 	display: inline-block;
 	vertical-align: middle;
 }
+
+/* =Hashtags
+-------------------------------------------------------------- */
+.liveblog-hash {
+	color: #55acee;
+	font-weight: bold;
+}
+
+.liveblog-hash::before {
+	content: "#";
+}
+
+/* =Authors
+-------------------------------------------------------------- */
+.liveblog-author {
+	font-weight: bold;
+}
+
+.liveblog-author::before {
+	content: "@";
+}
+
+/* =Emoji
+-------------------------------------------------------------- */
+.liveblog-emoji {
+	width: 18px;
+	height: 18px;
+}
+
+/* =Commands
+-------------------------------------------------------------- */
+.liveblog-command {
+	display:none;
+}
+
+/* =Key Events
+-------------------------------------------------------------- */
+
+.liveblog-key-events h2 {
+	color: #333;
+	margin-bottom: 15px;
+}
+
+.liveblog-key-events .liveblog-entry {
+	border: none;
+	overflow: visible;
+	padding: 0px;
+}
+
+.liveblog-key-events .liveblog-entry a:hover {
+	text-decoration: underline;
+}
+
+.liveblog-key-events li.liveblog-entry p {
+	margin-bottom: 0px;
+}
+
+.liveblog-key-events .liveblog-key-timeline li {
+	display: block;
+	list-style: none;
+}
+
+.liveblog-key-events .liveblog-key-timeline li .link {
+	position: relative;
+	display: block;
+	padding: 0 10px 20px;
+	border-left: 1px solid #dfdfdf;
+}
+
+.liveblog-key-events .liveblog-key-timeline li .link:before {
```

Comment:
@imsimond I’ve deleted those timeline styles and pushed up an alternative style (will still require some cross browser testing).